state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1540,6 +1542,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				destination_asset,
 				broker_fees,
 				refund_params,
+				dca_params,
```

Comment:
What is the exact plan to address this? A hotfix for 1.5 and a refactor here/separate PR?

Reply:
TBC...

Reply:
@msgmaxim 

Update: There's an open PR to main - but it might be simpler to just fix it here. LMK.

Since the fix will be applied to 1.5, no need for a migration.

Reply:
Cherry picked that PR. I also updated DCA migration code to include broker fee for ccm.